admin-front/src/components/TimeSeries/functions/getDataForExport.tsx

```diff
@@ -0,0 +1,72 @@
+import { TimeSeriesRecord, TimeSeriesType } from '../utility';
+import moment from 'moment';
+
+interface SeriesArrays {
+  priceVolume: TimeSeriesRecord[];
+  nftCount: TimeSeriesRecord[];
+}
+
+interface ExportParamObj {
+  timeseries: TimeSeriesRecord[];
+  context: TimeSeriesType;
+}
+
+type GetDataForExport = Array<ExportParamObj>;
+
+export const getDataForExport = (data: GetDataForExport) => {
+  const seriesArrays: SeriesArrays = { priceVolume: [], nftCount: [] };
+
+  data.forEach((exportObj) => {
+    const { context } = exportObj;
+
+    const exportKeyNames = {
+      timestamp: `${context}Date`,
+      value: `${context}Value`,
+    };
+
+    const contextArray = exportObj.timeseries.map((record: any) => {
+      record[exportKeyNames.timestamp] = moment
+        .unix(record['timestamp'])
+        .format('DD/MM/YY HH:MM');
+      record[exportKeyNames.value] = record['value'];
+      delete record['timestamp'];
+      delete record['value'];
+
+      return record;
+    });
+
+    seriesArrays[context] = contextArray;
+  });
+
+  let longestLength = 0;
+
+  for (let key in seriesArrays) {
+    if (seriesArrays.hasOwnProperty(key)) {
+      const lengthOfCurrentArray =
+        seriesArrays[key as keyof SeriesArrays].length;
+      if (lengthOfCurrentArray > longestLength) {
+        longestLength = lengthOfCurrentArray;
+      }
+    }
+  }
+
+  let exportArray: Array<Record<string, number>> = [];
+
+  if (longestLength > 0) {
```

Comment:
This `if` isn't needed. If longestLength is 0 or less than 0 (lets say there's a bug and less than 0 happens), the for loop will immediately break on entering (because `0 < 0` is false, and so is any 0 < x where x is a negative number)